.github/workflows/build_and_deploy.yml

```diff
@@ -148,16 +148,18 @@ jobs:
 
       - name: Build and push docker image from builder target
         uses: docker/build-push-action@v6
+        env:
+          RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
+          BUILDKIT_INLINE_CACHE: 1
```

Comment:
Was this `BUILDKIT_INLINE_CACHE` move needed? 
Our [docker docs](https://github.com/DFE-Digital/teaching-vacancies/blob/main/documentation/operations/infrastructure/docker.md#building-with-github-actions-workflow) (not written by me, so missing context here) explicitly states that this env var should be passed as a build-arg to the image.

Reply:
Thanks - it was a bit hit and hope to start with. I've removed some other bits too